### DISCUSSION TITLE
Сбивается фокус с ввода даты если рядом 2 поля дата

### DISCUSSION THREAD

**Discussion by simplepersonru** [totumonline/totum-ru-issues-and-discussions#382] [ОШБК!] [answered]
Если фокус стоит на втором поле дата, то при попытке переключиться на первое поле, фокус снова меняется на второе. Поставить фокус на первую дату можно только если предварительно убрать фокус с обоих дат. Возможно проблема глубже, чем только описанный мною сценарий, не исследовал

 
[Screencast from 21.05.2023 19:07:34.webm](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/71624171/5c468357-90cf-413d-a18f-2ad407eada79)

**Comment by ttm-alexeypolunin** [comment]
это известная ошибка

**Reply by ttm-alexeypolunin** [reply, marked as answer]
**вроде починили, но это неточно**
